app/views/drawings/index.html.haml

```diff
@@ -1,5 +1,2 @@
 #drawings
   = render 'drawings'
-
-#paginator.text-center
-  = link_to_next_page @drawings, 'MORE', remote: true, id: 'load_more'
```

Comment:
Are we sure we want to move pagination altogether? I'm hesitant as our data is image-based so eventually the homepage will get extremely long and turtle-like. 🐢 

We can def remove it for the pilot, and focus on how we want to paginate later. In that case, if we're removing it we should also remove the [JS partial that handles the pagination](https://github.com/empowerhack/DrawMyLife-Service/blob/master/app/views/drawings/index.js.erb)